src/NuGet.Core/NuGet.Shared/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "version": "3.5.0-*",
-  "shared": "*.cs",
+  "shared": "**/*.cs",

```

Comment:
That may explain some of the issues I was seeing with that package. I moved projects to just reference the files directly instead of using the shared package at all.